tests/cupy_tests/cuda_tests/test_memory.py

```diff
@@ -275,6 +275,18 @@ def test_alloc_split(self):
         self.assertEqual(ptr, head.ptr)
         self.assertEqual(ptr + self.unit * 2, tail.ptr)
 
+    def test_alloc_limit(self):
+        self.pool.set_limit(size=(self.unit * 6))
+
+        p1 = self.pool.malloc(self.unit * 5)
+        p2 = self.pool.malloc(self.unit * 1)
+        with self.assertRaises(memory.OutOfMemoryError):
+            self.pool.malloc(self.unit)
+
+        self.pool.set_limit(size=(self.unit * 7))
+        p3 = self.pool.malloc(self.unit)
+        del p1, p2, p3
```

Comment:
This is not needed.

Reply:
Without this line, they are warned by flake8 because they become unused variables.

Reply:
Hmm, I think we should use `# NOQA` for that purpose, but it's OK for consistency with other tests.